RPG-game-Hung/classes/magics.py

```diff
@@ -0,0 +1,110 @@
+import random
+from enum import Enum, auto
+
+from termcolor import colored
+
+from classes.colors import Color
+from classes.db import not_number
+
+
+class MagicType(Enum):
+    light = auto()
```

Comment:
for better readability maybe consider using `_generate_next_value_` as shown in [this example](https://docs.python.org/3/library/enum.html#using-automatic-values)